app/pages/intl/LangugeSwitcher.tsx

```diff
@@ -0,0 +1,33 @@
+import type { FunctionComponent } from 'react'
+import type { PropsWithTranslation, Translation, Translations } from '../../hocs/withTranslation'
+import { withTranslation } from '../../hocs/withTranslation'
+import { Language, useLanguage } from '../../hooks/useLanguage'
+
+interface LanguageSwitcherTranslation extends Translation {
+  toggleLanguage: string
+}
```

Comment:
`type` vs `interface` again.
Is there any practical use for the `extends Translation` here?
With the change suggested in `app/hocs/withTranslation.tsx` this would just be a type error if it didn't conform to the Translation type when used further below with `Translations<...>`.